hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHome.java

```diff
@@ -55,8 +54,16 @@
     public PhoneHome(Node node) {
         hazelcastNode = node;
         logger = hazelcastNode.getLogger(com.hazelcast.internal.util.phonehome.PhoneHome.class);
+        basePhoneHomeUrl = "http://phonehome.hazelcast.com/ping";
+    }
+
+    public PhoneHome(Node node, String baseurl) {
```

Comment:
Please make this constructor package-private. We use it only from the tests, so there is no need to expose it publicly.